src/main/java/io/vertx/json/schema/impl/SchemaValidatorImpl.java

```diff
@@ -599,7 +600,7 @@ private OutputUnit validate(final Object _instance, final JsonSchema schema, fin
                 thisEvaluated.add(key);
               } else {
                 stop = outputFormat == OutputFormat.Flag;
-                errors.add(new OutputUnit(subInstancePointer, computeAbsoluteKeywordLocation(schema, schemaLocation + "/patternProperties"), baseLocation + "/patternProperties", "Property \"" + key + "\" matches pattern \"" + pattern + "\" but does not match associated schema"));
+                errors.add(new OutputUnit(subInstancePointer, computeAbsoluteKeywordLocation(schema, schemaLocation + "/patternProperties"), baseLocation + "/patternProperties", "Property \"" + key + "\" matches pattern \"" + pattern + "\" but does not match associated schema", OutputErrorType.MISSING_VALUE));
```

Comment:
I think ig the property isn't matching the value, it is more a INVALID_VALUE than a MISSING_VALUE, what do you think?

Reply:
I think you are right here, I've updated it to be INVALID_VALUE now. Good catch thanks!